src/actions/actionCreators.js

```diff
@@ -447,3 +458,27 @@ export const setNewCategory = (category) => ({
   type: SET_NEW_CATEGORY,
   payload: { category },
 });
+
+export const setClaimVisibility = (claim, value) => ({
+  type: SET_CLAIM_VISIBILITY,
+  payload: { claim, value },
+});
+
+export const updateSelectedClaims = (claim) => ({
+  type: UPDATE_SELECTED_CLAIMS,
+  payload: { claim },
+});
+
+export const clearSelectedClaims = () => ({
+  type: CLEAR_SELECTED_CLAIMS,
```

Comment:
Maintain the footprint/signature of an action. Even if the payload is just an empty object, it should be there.